Attributed/UIFont.swift

```diff
@@ -32,33 +32,48 @@ extension UIFont {
     }
     
     func fontWithMonospacedNumbers() -> UIFont {
-        return UIFont(descriptor: fontDescriptor.addingAttributes([
-            UIFontDescriptorFeatureSettingsAttribute: [
-                [
-                    UIFontFeatureTypeIdentifierKey: kNumberSpacingType,
-                    UIFontFeatureSelectorIdentifierKey: kMonospacedNumbersSelector,
-                ],
-            ],
-            ]), size: pointSize)
+        #if os(watchOS)
```

Comment:
I tried variations of that and it failed. I'll try copy paste of what you have just to be sure...